openwisp_notifications/static/openwisp-notifications/js/notifications.js

```diff
@@ -238,7 +238,14 @@ function notificationWidget($) {
         }
         klass = notificationReadStatus.get(elem.id);
 
-        // Remove hyperlinks from notification messages if description is present
+        // If description is present, remove hyperlinks from the message
```

Comment:
The comment shouldn't be here but above the line which calls `replace`.
The part which says "if description is present" is redundant, the code is self explanatory and we don't need to add a comment that explains the if, we assume that anybody trying to read this code understands `if` statements.